sys-apps/systemd-utils: add 254.3

vimproved opened the pull request:
I assume these patches definitely will want to be added to the musl patchset, but I'm just putting this PR here as a source of patches and as a reference for when it does get bumped. systemd-boot 254 BDEPENDs on binutils for ld.bfd and RDEPENDs on gcc for libgcc (although it does seem to work with lld and compiler-rt, but I'm not putting that patch here.)

thesamesam commented on sys-apps/systemd-utils/files/systemd-utils-254_rc3-dt_relr.patch:
```diff
@@ -0,0 +1,26 @@
+https://bugs.gentoo.org/910570
```
You should be able to drop this, it's in 254.

vimproved replied:
Gotcha.

floppym commented:
I always use openembedded's patch set as a starting point for each systemd release.

https://git.openembedded.org/openembedded-core/tree/meta/recipes-core/systemd/systemd_254.bb#n37

Are any of your new patches included there already?

vimproved commented:
> I always use openembedded's patch set as a starting point for each systemd release.
> 
> https://git.openembedded.org/openembedded-core/tree/meta/recipes-core/systemd/systemd_254.bb#n37
> 
> Are any of your new patches included there already?

Didn't know that existed, I'll take a look at them and push an updated version.

vimproved commented:
Working on redoing this with openembedded's patches.

floppym commented:
Thanks. I've been meaning to bump this, but I keep forgetting about it.

vimproved commented:
Rebased on master and made to use openembedded's patches. Also, I moved the patchset to a git repo so the only thing that should be required when merging this is changing the SRC_URI for the musl patchset. Only two patches written by me, one of which is adapted from openembedded's systemd-boot fix (but needs to be a patch because of differences in build configuration -- openembedded builds systemd-boot separately so they append a flag that is problematic to the rest of systemd), and the other of which is a non-musl that adds an option to link kernel-install shared or statically (this should probably go upstream).
